modules/compiler/compiler_pipeline/source/pass_functions.cpp

```diff
@@ -60,8 +60,7 @@ uint64_t computeApproximatePrivateMemoryUsage(const llvm::Function &fn) {
     if (alloca_inst.isArrayAllocation()) {
       auto *arr_size_val = alloca_inst.getArraySize();
       auto *const_int = llvm::dyn_cast<llvm::ConstantInt>(arr_size_val);
-      assert(const_int != nullptr &&
-             "OpenCL or Vulkan Array Allocation of dynamic size");
+      assert(const_int != nullptr && "OpenCL Array Allocation of dynamic size");
```

Comment:
It was always unnecessary to include "OpenCL or Vulkan" here, could just be "Array allocation of dynamic size"